README.md

```diff
@@ -1,4 +1,6 @@
 # Nowa ZHP
+[![Build theme for WP sites](https://github.com/zhp-wlkp/zhp-pl/actions/workflows/build.yml/badge.svg)](https://github.com/zhp-wlkp/zhp-pl/actions/workflows/build.yml)
```

Comment:
Ten badge to powinien iść do głównego repo, a nie forka, ale to tam mogę poprawić po merge'u